ewatercycle/forcing_data.py

```diff
@@ -0,0 +1,9 @@
+class ForcingData(object):
```

Comment:
Maybe a dumb question, but why put this in a separate file? If I understand it correctly this leads to the following hierarchy (e.g. in the documentation)
```
ewatercycle 
- forcing
  - generate
- forcing_data
  - ForcingData
```

Wouldn't it make more sense to have something like:

```
ewatercycle
- forcing
  - ForcingData
  - generate
```
?

Reply:
`forcing.py` is already quite big, and I also expect `ForcingData` will not turn out to be a small class, so I put it in a different file. Besides that, it also gives some nice separation of concerns.

Later on we can move both files in to a sub directory to keep them together, i.e. `./ewatercycle/forcing/`

Reply:
Indeed, I like that last suggestion. Why not do that right away?

Reply:
Notice that Stefan also made a `models` module in #35

Reply:
Restructured the code a bit in the latest commit 👍